.scala-steward.conf

```diff
@@ -3,6 +3,7 @@ pullRequests.frequency = "@monthly"
 updates.pin  = [
   { groupId = "org.scalatest", artifactId = "scalatest", version = "3.1." }
   { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-core", version = "2.11." }
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-databind", version = "2.11." }
```

Comment:
The list may need to be longer:

https://github.com/lagom/lagom/blob/master/.scala-steward.conf#L12-L21

Reply:
ah, yes, this can be an ignore instead of a pin. Updated